masonry/src/widgets/flex.rs

```diff
@@ -1009,7 +1012,8 @@ impl Widget for Flex {
         // indicates that the box constrains for the following children have changed. Therefore they
         // have to calculate layout again.
         let bc_changed = self.old_bc != *bc;
-        let mut any_changed = bc_changed;
+        let mut any_changed = bc_changed || self.child_removed;
```

Comment:
Effectively, `any_changed` is being used a proxy for if the amount of "available flex space" has changed since last run, right?

This all feels very fragile.